docs/source-pytorch/cli/lightning_cli.rst

```diff
@@ -2,9 +2,25 @@
 
 .. _lightning-cli:
 
-############################
-Eliminate config boilerplate
```

Comment:
I preferred the previous title. 
To me, "Eliminate boilerplate" sounds more inviting to someone to try this feature.

Reply:
Similar to my previous comment. My guess is that people new to lightning will probably first search for hyperparameters and not reduction of boilerpate. In my opinion most important than the title was to combine the previous "Configure hyperparameters from the CLI" section with the one for LightningCLI so that when people go to it, there is clarity that they are both about implementing CLIs.

Reply:
I think the title here should be the same as how it is shown on the main menu. For subsections could be other titles. Would be good to hear more opinions about titles.

Reply:
What do other reviewers think? @carmocca @rohitgr7 @Borda @RobertLaurella

Reply:
cc: @lantiga